tests/meta/orders/test_meta_orders_api.py

```diff
@@ -38,7 +38,7 @@ def test_submit(base_store, orders_api: OrdersAPI, all_orders_to_submit, monkeyp
     order_ticket = {"name": "Paul Anderson", "email": "paul@magnolia.com"}
     result = orders_api.submit(order_type, data=order_data, ticket=order_ticket)
 
-    # THEN it should work...
+    # THEN it should work
```

Comment:
What should work? 😄

Reply:
a bit of a poor description huh? I don't think I have invented this test, I'll have a look to see if I can come up with something better

Reply:
fixed